bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueBridgeDiscoveryParticipant.java

```diff
@@ -92,4 +113,20 @@
         }
         return null;
     }
+
+    @Override
+    public long getRemovalGracePeriodSeconds(RemoteDevice device) {
+        try {
+            Configuration conf = configAdmin.getConfiguration("binding.hue");
+            Dictionary<String, @Nullable Object> properties = conf.getProperties();
```

Comment:
Looks.like we need an additional null-check on `properties` before accessing them. See https://community.openhab.org/t/org-openhab-binding-hue-internal-discovery-huebridgediscoveryparticipant-threw-an-exception/117247

Reply:
New PR created #10199